### ISSUE TITLE
NullPointerException

### ISSUE THREAD

**Issue by nibogd** [fabianonline/telegram_backup#30]
I've got an exception after half of the messages were downloaded, now i'm getting it on every start. Is there a way to get the message by ID (just to see what makes it fail)?

```
Downloading 224718 messages.
15:03:57 DEBUG DownloadManager.downloadMessages   : Entering download loop
15:03:57 TRACE DownloadManager.downloadMessages   : Loop
15:03:57 TRACE DownloadManager.downloadMessages   : vector.size(): 100
15:03:57 TRACE DownloadManager.downloadMessages   : ids.size(): 224618
...100java.lang.NullPointerException
    at de.fabianonline.telegram_backup.mediafilemanager.DocumentFileManager.isSticker(DocumentFileManager.java:63)
    at de.fabianonline.telegram_backup.mediafilemanager.FileManagerFactory.getFileManager(FileManagerFactory.java:54)
    at de.fabianonline.telegram_backup.Database.saveMessages(Database.java:219)
    at de.fabianonline.telegram_backup.DownloadManager.downloadMessages(DownloadManager.java:183)
    at de.fabianonline.telegram_backup.DownloadManager._downloadMessages(DownloadManager.java:135)
    at de.fabianonline.telegram_backup.DownloadManager.downloadMessages(DownloadManager.java:74)
    at de.fabianonline.telegram_backup.CommandLineController.<init>(CommandLineController.java:165)
    at de.fabianonline.telegram_backup.CommandLineRunner.main(CommandLineRunner.java:77)
java.lang.RuntimeException: Exception shown above happened.
    at de.fabianonline.telegram_backup.Database.saveMessages(Database.java:281)
    at de.fabianonline.telegram_backup.DownloadManager.downloadMessages(DownloadManager.java:183)
    at de.fabianonline.telegram_backup.DownloadManager._downloadMessages(DownloadManager.java:135)
    at de.fabianonline.telegram_backup.DownloadManager.downloadMessages(DownloadManager.java:74)
    at de.fabianonline.telegram_backup.CommandLineController.<init>(CommandLineController.java:165)
    at de.fabianonline.telegram_backup.CommandLineRunner.main(CommandLineRunner.java:77)

----- EXIT -----
```

**Comment by Rulipoke** [comment]
I have the exact same problem.

```
Downloading most recent dialogs...
Top message ID is 1785261
Top message ID in database is 844203
Downloading 941058 messages.
...200java.lang.NullPointerException
        at de.fabianonline.telegram_backup.mediafilemanager.DocumentFileManager.isSticker(DocumentFileManager.java:63)
        at de.fabianonline.telegram_backup.mediafilemanager.FileManagerFactory.getFileManager(FileManagerFactory.java:54)
        at de.fabianonline.telegram_backup.Database.saveMessages(Database.java:219)
        at de.fabianonline.telegram_backup.DownloadManager.downloadMessages(DownloadManager.java:208)
        at de.fabianonline.telegram_backup.DownloadManager._downloadMessages(DownloadManager.java:136)
        at de.fabianonline.telegram_backup.DownloadManager.downloadMessages(DownloadManager.java:75)
        at de.fabianonline.telegram_backup.CommandLineController.<init>(CommandLineController.java:168)
        at de.fabianonline.telegram_backup.CommandLineRunner.main(CommandLineRunner.java:77)
java.lang.RuntimeException: Exception shown above happened.
        at de.fabianonline.telegram_backup.Database.saveMessages(Database.java:281)
        at de.fabianonline.telegram_backup.DownloadManager.downloadMessages(DownloadManager.java:208)
        at de.fabianonline.telegram_backup.DownloadManager._downloadMessages(DownloadManager.java:136)
        at de.fabianonline.telegram_backup.DownloadManager.downloadMessages(DownloadManager.java:75)
        at de.fabianonline.telegram_backup.CommandLineController.<init>(CommandLineController.java:168)
        at de.fabianonline.telegram_backup.CommandLineRunner.main(CommandLineRunner.java:77)

----- EXIT -----
```

**Comment by imagest** [comment]
This looks like it's checking for docs variable. I imagine some sort of edge case.

I fixed this by adding this line.
        if (doc == null) return false;

**Comment by fabianonline** [comment]
Thanks for the info and also the code snippet. It's fixed in 1.0.6 now.
